source/guides/views/layouts.md

```diff
@@ -66,6 +66,37 @@ end
 
 If we want to turn off this feature entirely, we can set `layout nil` into the application's configuration.
 
+## Using Multiple Templates
+
+Sometimes it's useful to have more than one template. For example, if the `application.erb` template contains an HTML structure you don't want for, say, a login page - a separate `login.html.erb` template can be used.
+
+Assuming you have created a `home#login` action, you create the `login.html.erb` template right next to your default `application.html.erb` in `/web/templates/`.
+
+Hanami doesn't know about the existence of a `Web::Views::LoginLayout` yet, so make sure to create a `login_layout.rb` right next to default `application_layout.rb` in `/views`. Inside, you pretty much copy default, changing only the class name:
```

Comment:
Don't need to say `Inside, you pretty much copy default, changing only the class name`, since I think people will be more likely to copy and paste from the code below than copy from their existing layout class.